src/components_redux/Radios.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+
+export default class Radios extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      type: "text"
+    };
+    const options = {
+      text: "Textfield",
+      radios: "Radios",
+      checkbox: "CheckBox",
+      select: "Select",
+      name: "input",
+      type: "radio"
+    };
+  }
+  radioChange = event => {
+    this.setState({
+      type: event.target.value
+    });
+  };
+  render() {
+    return (
+      <div>
+        <label>{this.props.title}</label>
+        
+        <label>
```

Comment:
After you finished with the loop here, you will notice that you need now the selected value in the parent component (Input.js) to be able to change the `showComponent` in Input depneding on the selection.

What you need for this is a handler property, which will be called with the new value. Similar to `onChange` for input elements. So you will need another property for Radios `onChange` which is a callback function. Inside `radioChange` you then can call `onChange` with the selected value. In Input you receive the value and depending on the value you can set `showComponent`.

Reply:
Resolved in new commit.